rlbot_gui/story/bots.json

```diff
@@ -0,0 +1,65 @@
+{
+    "psyonix-pro": {
+        "name": "Psyonix Pro",
+        "type": "psyonix",
+        "skill": 0.5
+    },
+    "psyonix-allstar-name": {
+        "name": "Psyonix Allstar",
+        "type": "psyonix",
+        "skill": 1
+    },
+    "adversity": {
+        "name": "AdversityBot",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "ReliefBotFamily", "README", "adversity_bot.cfg"]
+    },
+    "airbud": {
+        "name": "Airbud",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "ReliefBotFamily", "README", "air_bud.cfg"]
+    },
+    "reliefbot": {
+        "name": "ReliefBot",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "ReliefBotFamily", "README", "relief_bot.cfg"]
+    },
+    "sdc": {
+        "name": "Self-Driving Car",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "Self-driving car", "self-driving-car.cfg"]
+    },
+    "kamael": {
+        "name": "Kamael",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "Kamael", "Kamael.cfg"]
+    },
+    "botimus": {
+        "name": "BoltimusPrime",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "BotimusPrime", "botimus.cfg"]
+    },
+    "bumblebee": {
+        "name": "Bumblebee",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "BotimusPrime", "bumblebee.cfg"]
+    },
+    "baf": {
+        "name": "Missle - Heat Seeking Kind",
+        "type": "rlbot",
+        "skill": 1,
+        "path": ["$RLBOTPACKROOT", "RLBotPack", "blind_and_deaf", "bot.cfg"]
+    },
```

Comment:
Why rename the bot? Not a fan of the name "blind and deaf"?

Reply:
I was worried it came off as a bit insensitive. Though maybe I was overthinking it. I have named it BAF for now to keep it closer to the original name.

Reply:
Cool, makes sense.